### DISCUSSION TITLE
offload detector to OpenGL GPU on LibrePi

### DISCUSSION THREAD

**Discussion by Phatkone** [blakeblackshear/frigate#9268] [Ask A Question]
I have a LibrePi Renegade (ROC-RK3328-CM), purchased because it has OpenGL/OpenVG and I thought that would be good for offloading the detection processing.  
I have been reading through the docs and not seeing anything that matches for detector types.  
  
Anyone know if this is currently possible? if so, which decoder type do I use?  
  
Any help is greatly appreciated.  
  
Cheers

**Comment by NickM-27** [comment]
Only one that might be compatible is https://deploy-preview-6262--frigate-docs.netlify.app/configuration/object_detectors#rockchip-rknn-toolkit-lite2

**Reply by Phatkone** [reply]
thanks for that. I cant see the RKNPU when i tried to check the load ($ cat /sys/kernel/debug/rknpu/load) 
so perhaps its different on my model libre :/

Suspect its older documentation there too, because i could not see a "-rk" docker image in docker hub and the normal one does not accept that detector type

Bugger.
